src/test/java/test/thread/parallelization/BaseParallelizationTest.java

```diff
@@ -560,14 +561,26 @@ public static void verifyEventsForTestMethodsRunInTheSameThread(Class<?> testCla
             testName) {
 
         for (Method method : testClass.getMethods()) {
-            if (method.getDeclaringClass().equals(testClass)) {
+            boolean isTestMethod = false;
```

Comment:
As for coverage:

I am about 40 percent done for parallel-by-methods tests. I am about to start covering the effect of dependencies on parallelization by methods, but these test classes should be must faster to write because of all of the work I did in the base parallelization test class for the first 8 test cases. 

Following that I need to do similar tests for:

Parallelization by tests
Parallelization by class
Parallelization by instance
Parallel data providers

So overall -- about 10% coverage of the entire feature.

Most of the bugs will surface in the tests that involve dependencies.  Another contributor just did some work in the graph logic related to dependencies, so some of the parallelization defects might have been fixed by that.